tests/extensions/plugins/test_webhook.py

```diff
@@ -40,6 +46,66 @@ def test_trigger_webhooks_for_event(
     )
 
 
+first_url = "http://www.example.com/first/"
+third_url = "http://www.example.com/third/"
+
+
+@pytest.mark.parametrize(
+    "event_name, total_webhook_calls, expected_target_urls",
+    [
+        (WebhookEventType.PRODUCT_CREATED, 1, {first_url}),
+        (WebhookEventType.ORDER_FULLY_PAID, 2, {first_url, third_url}),
+        (WebhookEventType.ORDER_FULFILLED, 1, {third_url}),
+        (WebhookEventType.ORDER_CANCELLED, 1, {third_url}),
+        (WebhookEventType.ORDER_UPDATED, 1, {third_url}),
+        (WebhookEventType.ORDER_CREATED, 1, {third_url}),
+        (WebhookEventType.CUSTOMER_CREATED, 0, set()),
+    ],
+)
+@mock.patch("saleor.extensions.plugins.webhook.tasks.send_webhook_request.delay")
+def test_trigger_webhooks_for_event_calls_expected_events(
+    mock_request,
+    event_name,
+    total_webhook_calls,
+    expected_target_urls,
+    service_account,
+    order_with_lines,
+    permission_manage_orders,
+    permission_manage_users,
+    permission_manage_products,
+):
+    service_account.permissions.add(permission_manage_orders)
```

Comment:
It tests the permissions, right? Could we precise it? In the docstring or somewhere so we know what we are testing.

Reply:
As the name says - `trigger_webhooks_for_event` calls expected events. I don't know what else should I add to docstring